luigi/parameter.py

```diff
@@ -331,9 +337,35 @@ class MonthParameter(DateParameter):
 
     date_format = '%Y-%m'
 
-    @staticmethod
-    def next_in_enumeration(_value):
-        return None
+    def _add_months(self, date, months):
+        """
+        Add ``months`` months to ``date``.
+
+        Unfortunately we can't use timedeltas to add months because timedelta counts in days
+        and there's no foolproof way to add N months in days without counting the number of
+        days per month.
+        """
+        year = date.year + (months // 12)
```

Comment:
this logic seems complicated. pretty sure you can do it using something like this

`year = date.year + (months-1)//12`
`month = date.month + (months-1)%12 + 1`